File: id/src/main/java/com/simprints/id/sync/models/RealmSyncInfo.java
package com.simprints.id.sync.models;

import java.util.Date;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class RealmSyncInfo extends RealmObject {
    @PrimaryKey
    public int id;
    public Date lastSyncTime;

    public RealmSyncInfo() {
    }

    public RealmSyncInfo(Date lastSyncTime) {
        this.id = 0;
        this.lastSyncTime = lastSyncTime;
    }
}

File: id/src/main/java/com/simprints/id/sync/models/RealmSyncInfo.kt
package com.simprints.id.sync.models

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import java.util.*

open class RealmSyncInfo(@field:PrimaryKey var id: Int = 0,
                         var lastSyncTime: Date = Date(0)) : RealmObject()
